README.md

```diff
@@ -146,3 +146,9 @@ Rarely but the repo admins might need to check in changes to the repo that are
 not AOD request. To do that, the admin would need to send PRs as usual with
 "lock branch" temporarily disabled. This activity should be coordinated
 carefully to make sure AOD request PRs won't be merged accidentally.
+
```

Comment:
Is there a better way than this? Like using breakglass to bypass when needed?

```
Rarely but the repo admins might need to check in changes to the repo that are
not AOD request. To do that, the admin would need to send PRs as usual with
"lock branch" temporarily disabled. This activity should be coordinated
carefully to make sure AOD request PRs won't be merged accidentally.```

Reply:
I am not sure about the breakglass mechanism people will use outside of our org, but added bypass for them to check out. Thanks!